### PR TITLE
add associations class files and fixed linters

### PR DESCRIPTION
**Pull request by diegoh40**
In this project, I finished creating the remaining classes for my school library and created the associations between them.

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -1,13 +1,15 @@
 require './nameable'
 require './capitalize'
 require './trimmer'
+
```
Comment (tufoinnkuo10) [inline person.rb:4]: - [ ] please ensure Getters and Setters for @rentals should be present
`attr_accessor :name, :age, :rentals`

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @diegoh40 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct GitHub flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by diegoh40** [comment]
The changes were implemented

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Diego, 
## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**